pypeit/edgetrace.py

```diff
@@ -787,6 +799,12 @@ def auto_trace(self, bpm=None, debug=False, show_stages=False):
         if show_stages:
             self.show(title='After synchronizing left-right traces into slits')
 
+        # [DP] `maskdesign_matching` for now is only matching the traces found with the ones predicted by
+        # the slit-mask design. If we include the piece of code in which `maskdesign_matching` recovers
+        # missing traces, we should move the following 2 lines before `self.sync()`
+        if maskdesign is True:
+            self.maskdesign_matching(offsets_range=[-50, 50], step=1, debug=False)
```

Comment:
I would add the parset parameters now, as otherwise we are breaking our design rules about spectrograph specific hooks in code (see my point above about calibrations).

Reply:
Added to the parset parameters in `EdgeTracePar`